test/src/core/stdlib.js

```diff
@@ -893,7 +897,7 @@ describe('foam.Object', function() {
 
 });
 
-describe('foam.mmethod', function() {
+describe('foam.method', function() {
```

Comment:
Maybe you thought `mmethod` was a typo? That's the actual name of the method being tested though, it contains two m's. Unless I'm missing something here.

Reply:
Oh yeah, you're right.